### ISSUE TITLE
Still unable to connect to server, conenction closes

### ISSUE THREAD

**Issue by DenialRiver1434** [cpinitiative/ide#102]
Connection still closed all the time for me, I have tried on multiple devices using multiple networks.

![image](https://github.com/cpinitiative/ide/assets/52391257/9f5cbac8-5e25-4172-8582-ef882dec6e77)

**Comment by thecodingwizard** [comment]
Can you check the console for error messages? Unfortunately, I can't reproduce this anymore :(

**Comment by S-D-Ge** [comment]
The IDE is working again on my home laptop, but it still doesn't work on my school laptop (it had worked for most of the year). 

The two highlighted error messages appear on my when I try to run code (the other error messages appear on my home computer as well so they're probably related to this issue).

![image](https://github.com/cpinitiative/ide/assets/52391257/ac253614-7f8b-4262-9533-f837cb5842bf)

This seems to be an issue with my school's firewall, do you know which site I can request them to unblock?

**Comment by thecodingwizard** [comment]
I don't think it's an issue with your school firewall. For some reason, you seem to be using a cached / outdated version of the USACO Guide IDE. Perhaps try incognito, a different browser, disable cache, ...?

**Comment by S-D-Ge** [comment]
That's strange, it shows up as ggzk2rm2ad whether I use ide.usaco.guide or beta.ide.usaco.guide, I can test it on some of my friend's computers tomorrow and update how it goes.

**Comment by thecodingwizard** [comment]
oh wait, sorry -- I was mistaken. ggzk2rm2ad is correct. Maybe it is a school firewall issue.

On the computer / network having this issue, if you visit https://ggzk2rm2ad.execute-api.us-west-1.amazonaws.com/Prod/execute, do you see a "message: missing authentication token" response?

**Comment by S-D-Ge** [comment]
Indeed, it is blocked and I have filled out an unblock request. Thanks.

![image](https://github.com/cpinitiative/ide/assets/52391257/8ca0ef18-8e46-49c4-a115-eb12e9169e77)

**Comment by thecodingwizard** [comment]
ah, perfect. thanks for investigating this! I'll close this issue now; please feel free to re-open if another issue comes up.
